CONTRIBUTING.md

```diff
@@ -35,15 +36,23 @@ that the description is in Markdown, so reformat from Rst if necessary).
 Tag the release according to [semantic versioning
 guidelines](https://semver.org/), preceded with a `v` (e.g.: v1.0.0). The
 release title is the tag and the release date together (e.g.: v1.0.0
-(2019-07-25)). Tick the pre-release box in case the release is a candidate
-release, and amend the version tag with `rc` and the candidate number.
+(2019-07-25)).
+
+### Release candidate
+When releasing a release candidate on Github, tick the pre-release box, and
+amend the version tag with `rc` and the candidate number. Ensure the release
```

Comment:
To follow [semantic versioning](https://semver.org/#spec-item-9) formatting it should be 
```suggestion
amend the version tag with `-rc` and the candidate number. Ensure the release
```